css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
```

Comment:
It is a good thing to first overwrite all the margin, padding and box sizing at the top of your css.

* { padding: 0; margin: 0; box-sizing: border-box; }